lib/RegisterController.js

```diff
@@ -71,3 +78,29 @@ export class RegisterController {
     }
   }
 }
+
+async function _verifyTurnstileToken({token}) {
+  // dummy secret key, load actual key from config
+  const SECRET_KEY = '1x0000000000000000000000000000000AA';
+  // TODO: load url from config
+  const url = 'https://challenges.cloudflare.com/turnstile/v0/siteverify';
+  const payload = {
+    secret: SECRET_KEY,
+    response: token,
+    // how to get remoteIP?
+    remoteip: ''
```

Comment:
Their example code uses Cloudflare Workers which is given the `remoteIP` in a Cloudflare provided header: https://github.com/cloudflare/turnstile-demo-workers/blob/main/src/index.mjs#L12

We'd need a server-side endpoint (Worker or otherwise) to get the IP address as client-side JS doesn't have access to it.